src/hooks/use-ordered-rows.ts

```diff
@@ -0,0 +1,23 @@
+import { useMemo } from 'preact/hooks';
+
+import type { Order } from '../types';
+
+export function useOrderedRows<Row>(rows: Row[], order?: Order<keyof Row>) {
```

Comment:
A return type annotation here would be helpful for the reader. Can you also add a short JSDoc comment explaining usage and behavior.